src/service/overseas.entities.service.ts

```diff
@@ -55,3 +57,29 @@ export const updateOverseasEntity = async (req: Request, session: Session) => {
 
   logger.debugRequest(req, `Updated Overseas Entity, ${JSON.stringify(response)}`);
 };
+
+export const resumeOverseasEntity = async (
```

Comment:
i'd call this function getOverseasEntity, it will be consistent then with the create and update functions in this file - and update the log messages to say 'get' instead of 'resume'

Reply:
Will do